packages/terra-table/src/proptypes/rowShape.js

```diff
@@ -0,0 +1,22 @@
+import PropTypes from 'prop-types';
+import cellShape from './cellShape';
+
+const rowShape = PropTypes.shape({
+  /**
+   * An identifier to uniquely identify the row within the table.
+   */
+  id: PropTypes.string.isRequired,
+
+  /**
+   * Data to be displayed in the cells of the row. Cells will be rendered in the row in the order given.
+   */
+  cells: PropTypes.arrayOf(cellShape),
+  /**
+   * A string identifier used to describe the row contents. This value will be used to construct additional labels
+   * for internal controls (e.g. row selection cells).
+   */
+  ariaLabel: PropTypes.string,
```

Comment:
The rowShape object does not have all of its props from the DataGrid implementation.